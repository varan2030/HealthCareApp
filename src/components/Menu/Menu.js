import React, { Component } from 'react';

class Menu extends Component {
  render() {
    return (
      <div className={ this.props.className}>
        { this.props.children }
      </div>
    )    
  }
}

export default Menu