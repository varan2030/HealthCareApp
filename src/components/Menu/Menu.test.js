import React from 'react';
import Enzyme, {shallow, mount} from 'enzyme';
import Menu from "./";
import Adapter from 'enzyme-adapter-react-16';

Enzyme.configure({adapter: new Adapter()});

describe("Menu component", () =>{
  
  test('renders', () => {
    const wrapper = shallow(<Menu />);
    expect(wrapper.exists()).toBe(true);
  })

  test('when the Menu is clicked the Modal is opened', () =>{
    const wrapper = (<Menu />);
    expect(wrapper).toMatchSnapshot();
  })
});
