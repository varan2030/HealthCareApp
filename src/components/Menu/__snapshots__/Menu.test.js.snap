// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Menu component when the Menu is clicked the Modal is opened 1`] = `<Menu />`;
